bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/i18n/LocaleProvider.java

```diff
@@ -25,4 +28,10 @@
      */
     Locale getLocale();
 
+    /**
+     * Provides access to the location of the installation
+     *
+     * @return location of the current installation or null if the location is not set
+     */
+    PointType getLocation();
```

Comment:
> Another consideration has to be what is "configured together", i.e. what should have an entry in ConfigAdmin and also appear as a single service e.g. in the Paper UI Configuration->Services section. Here as well, I think it makes sense to have those settings together in an "i18n" section. Splitting it in smaller pieces will only make it harder to find stuff.

Hm, isn't the configuration winded to a service.pid and the service pid is choosen by the implementation?
Can't we have different provider interfaces and one implementation that uses one configuration?

E.g. `I18nProviderImpl` implements the `LocaleProvider`, `TranslationProvider` and `LocationProvider` interface and is configured using the service pid `org.eclipse.smarthome.i18n`?